httpx/_api.py

```diff
@@ -56,6 +58,8 @@ def request(
     request.
     * **auth** - *(optional)* An authentication class to use when sending the
     request.
+    * **proxies** - *(optional)* A dictionary mapping HTTP protocols to proxy
```

Comment:
Perhaps "mapping proxy keys to proxy URLs"? Proxy routing supports more than just routing by HTTP protocol.